caddyhttp/pipa/validAndParse.go

```diff
@@ -0,0 +1,73 @@
+package pipa
+
+import (
+	"encoding/base64"
+	. "github.com/journeymidnight/yig-front-caddy/caddyerrors"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"unicode/utf8"
+)
+
+var fuc = []string{"resize", "watermark"}
```

Comment:
Use constant, and every words need be transfered to lowercase

Reply:
There are some doubts, I only called it once here, I don't feel the need to use constants